Swiftfin/Views/ItemEditorView/EditMetadataView/Components/Sections/OverviewSection.swift

```diff
@@ -0,0 +1,55 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2024 Jellyfin & Jellyfin Contributors
+//
+
+import Combine
+import JellyfinAPI
+import SwiftUI
+
+extension EditMetadataView {
+
+    struct OverviewSection: View {
+
+        @Binding
+        var item: BaseItemDto
+
+        let itemType: BaseItemKind
+
+        private var showTaglines: Bool {
+            [
+                BaseItemKind.movie,
+                .series,
+                .audioBook,
+                .book,
+                .audio,
+            ].contains(itemType)
+        }
+
+        var body: some View {
+            if showTaglines {
+                // There doesn't seem to be a usage anywhere of more than 1 tagline?
+                Section(L10n.taglines) {
+                    TextField(
+                        L10n.tagline,
+                        value: $item.taglines
+                            .map(
+                                getter: { $0 == nil ? "" : $0!.first },
+                                setter: { $0 == nil ? [] : [$0!] }
+                            ),
+                        format: .nilIfEmptyString
+                    )
+                }
+            }
+
+            // TODO: Size Up / Down with Text
+            Section(L10n.overview) {
+                TextEditor(text: $item.overview.coalesce(""))
+                    .frame(minHeight: 100, maxHeight: .infinity)
```

Comment:
@LePips do you know if there is a good way change the text field size based on content? It's likely fine to have as a set multi-line textField but I thought I'd double check there isnt a more dynamic way.

Reply:
This changed in height for me as I was editing it and we don't even need the `frame`, at least on the iOS 18 simulator.

Reply:
Without the frame, this is what this looks like (at least on iPad)

![Simulator Screenshot - iPad Pro 13-inch (M4) - 2024-11-27 at 12 00 08](https://github.com/user-attachments/assets/21c7a1a9-fbec-405f-b977-d459c153496b)

Reply:
Okay, I found this happening on iOS 17 and earlier at least with my testing. I found the overview would resize when edited, so setting the overview with itself looks like it triggers a resize.

[`6f3f1d2` (#1323)](https://github.com/jellyfin/Swiftfin/pull/1323/commits/6f3f1d237df7d7349a9d141099249b49ff8f8fa8)